_layouts/home.html

```diff
@@ -6,8 +6,11 @@
     <body>
         {% include navbar.html %}
         <div id="home-content" class="container">
-            
+
+            {% comment %}
```

Comment:
Is this intended as a TODO to remove after the event?

Reply:
Yes. I've preferred to leave it like that in case we want to add it back we already know where in the page it looked ok.